_run/single/README.md

```diff
@@ -85,7 +85,7 @@ use `ovrclk/akash:latest` to use the default image.
 
 __t1__
 ```sh
-DOCKER_IMAGE=ovrclk/akash:latest make kind-configure-image
+DOCKER_IMAGE=ovrclk/akash:stable make kind-configure-image
```

Comment:
I think we should update this to the ghcr.io package, right @adam?

Reply:
Yeah, let's.  I'll merge my PR and do a release to `v0.11.0-rc0` too so that we can use `:latest` here too.

Reply:
updated to use ghcr.io package